diff --git a/includes/VisualEditorHooks.php b/includes/VisualEditorHooks.php
index 1c81b1e..d9a4e39 100644
--- a/includes/VisualEditorHooks.php
+++ b/includes/VisualEditorHooks.php
@@ -405,6 +405,16 @@ class VisualEditorHooks {
 		// change "Edit" to "Edit source".
 		$isAvailable = $namespaceEnabled && $contentModelEnabled;
 
+		var_dump( [ 'DEBUG' => [
+			'title' => $title->getFullText(),
+			'namespaceEnabled' => $namespaceEnabled ? 'yes' : 'no',
+			'contentModel' => $pageContentModel,
+			'contentModelEnabled' => $contentModelEnabled ? 'yes' : 'no',
+			'isAvailable' => $isAvailable ? 'yes' : 'no',
+			'wgVisualEditorUseSingleEditTab' => $config->get( 'VisualEditorUseSingleEditTab' ),
+			'User preference' => $user->getOption( 'visualeditor-tabs' )
+		] ] );
+
 		// HACK: Exit if we're in the Education Program namespace (even though it's content)
 		if ( defined( 'EP_NS' ) && $title->inNamespace( EP_NS ) ) {
 			return true;
