diff --git a/includes/media/FormatMetadata.php b/includes/media/FormatMetadata.php
index 52d7373aab..9ff94d192b 100644
--- a/includes/media/FormatMetadata.php
+++ b/includes/media/FormatMetadata.php
@@ -1861,7 +1861,7 @@ class FormatMetadata extends ContextSource {
 		// be used so we take the easy way
 		$key = preg_replace( '/[^a-zA-z0-9_:.-]/', '', $key );
 		// drop characters which are invalid at the first position
-		$key = preg_replace( '/^[\d-.]+/', '', $key );
+		$key = preg_replace( '/^[\d\-.]+/', '', $key );
 
 		if ( $key == '' ) {
 			$key = '_';
diff --git a/includes/upload/UploadStash.php b/includes/upload/UploadStash.php
index e25b11ec23..e55ab1fbb7 100644
--- a/includes/upload/UploadStash.php
+++ b/includes/upload/UploadStash.php
@@ -52,7 +52,7 @@
  */
 class UploadStash {
 	// Format of the key for files -- has to be suitable as a filename itself (e.g. ab12cd34ef.jpg)
-	const KEY_FORMAT_REGEX = '/^[\w-\.]+\.\w*$/';
+	const KEY_FORMAT_REGEX = '/^[\w\-\.]+\.\w*$/';
 	const MAX_US_PROPS_SIZE = 65535;
 
 	/**
