diff --git a/tests/phpunit/framework/engine/cli/InvokedWikiSettings.php b/tests/phpunit/framework/engine/cli/InvokedWikiSettings.php
index d553f27..4b36ac3 100644
--- a/tests/phpunit/framework/engine/cli/InvokedWikiSettings.php
+++ b/tests/phpunit/framework/engine/cli/InvokedWikiSettings.php
@@ -41,6 +41,11 @@ function efModerationTestsuiteSetup() {
 			$response = RequestContext::getMain()->getRequest()->response();
 			$response->header( $string, $replace, $http_response_code );
 		}, true );
+
+		/* Disable header_register_callback(), causes segfault in uopz */
+		uopz_set_return( 'header_register_callback', function( callable $unused ) {
+			/* Nothing to do */
+		}, true );
 	}
 
 	/*
