diff --git a/extensions/Moderation/modules/ext.moderation.ajaxhook.js b/extensions/Moderation/modules/ext.moderation.ajaxhook.js
index e16d7d3..bbd734a 100644
--- a/extensions/Moderation/modules/ext.moderation.ajaxhook.js
+++ b/extensions/Moderation/modules/ext.moderation.ajaxhook.js
@@ -88,6 +88,7 @@
 		@returns New API response (if overwrite is needed) or false (if no need to overwrite).
 	*/
 	function rewriteAjaxResponse( query, ret ) {
+		console.log( "in rewriteAjaxResponse(): ", JSON.stringify( query ), JSON.stringify( ret ) );

 		/* Check whether we need to overwrite this AJAX response or not */
 		if ( !ret.error ) {
